### PR TITLE
Support argument to set random seed

### PR DESCRIPTION
**Pull request by bbimber**
My python is not great, but I think this is reasonable. See #4. I believe since the default value of the argument is None, if no value is set it should preserve the default behavior of today (https://scikit-learn.org/stable/modules/generated/sklearn.mixture.GaussianMixture.html)

### CODE REVIEW

**Comment by bbimber** [comment]
@xhongyi hey - i'm not sure if you saw this, but I think this is a workable solution to allow setting random seed from the command line. my python is not great, so it's possible there's a more elegant solution.

**Comment by bbimber** [comment]
@xhongyi just wanted to follow up. on #4 you sounded positive on this feature - anything I can do to help finalize this?

**Comment by bbimber** [comment]
@xhongyi just wanted to check back again - i'm happy to make changes or updates if there is anything about this that's a concern or if you think another approach is better. thanks.

**Comment by bbimber** [comment]
@xhongyi Just wanted to check back in - anything I can do to help move this forward?

**Comment by bbimber** [comment]
@xhongyi hey - just checking back in. I would prefer to contribute to the main project, but pretty soon i think i will end up forking it so we can take advantage of this.
